src/datajunction/models/node.py

```diff
@@ -167,7 +167,7 @@ def extra_validation(self) -> None:
         Extra validation for node data.
         """
         if self.type == NodeType.SOURCE:
-            if self.expression:
+            if self.query:
```

Comment:
Fixed this after noticing it was causing some test failures, looks like from a very recent change. There's a similar line change in `node_test.py`.

Reply:
Fixed 172 👍